selenium_tests/cross_listing/page_objects/cross_listing_base_page_object.py

```diff
@@ -0,0 +1,34 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+from selenium_common.base_page_object import BasePageObject
+
+
+class Locators(object):
+    @classmethod
+    def TD_TEXT_XPATH(cls, search_text):
```

Comment:
Ditto comment below, will give it a shot move and into base page object for this round.